crates/librqbit/src/tracing_subscriber_config_utils.rs

```diff
@@ -107,35 +114,43 @@ pub fn init_logging(opts: InitLoggingOptions) -> anyhow::Result<InitLoggingResul
                 .open(&log_file)
                 .with_context(|| format!("error opening log file {:?}", log_file))?,
         ));
-        layered
-            .with(
+        let log_env_filter = EnvFilter::builder()
+            .parse(opts.log_file_rust_log.unwrap_or("info,librqbit=debug"))
+            .context("can't parse log-file-rust-log")?;
+        let log_layer: Box<dyn Layer<_> + Send + Sync> = if opts.log_file_json {
+            Box::new(
                 fmt::layer()
-                    .with_ansi(false)
```

Comment:
this makes log file contain ANSI escape codes which makes it hard to grep and use. Those escape codes are intended for terminal use only.

PR LGTM otherwise, I'd merge if not for this